static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -131,7 +131,11 @@ const getMessageFromBraze = async (apiKey, brazeUuid) => {
     appboy.changeUser(brazeUuid);
     appboy.openSession();
 
-    const canShowPromise = brazeMessages.getMessageForBanner().then((m) => {
+    const brazeArticleContext = {
+        section: config.get('page.section')
+    };
```

Comment:
It looks like we're creating `brazeArticleContext` further down and passing to `getMessageFromBraze`, but then re-defining here? I think we only need to do one or the other. Unless I'm missing something?